docs/getting-started/package-infrastructure.md

```diff
@@ -0,0 +1,878 @@
+---
+title: Package Infrastructure
+toc: true
+weight: 6
+indent: true
+---
+
+# Package Infrastructure
+
+In a [previous section] we learned that Crossplane can be configured with new
+composite resources (XRs) that are [composed] of other resources, allowing you
+to define and offer resources that group and abstract infrastructure primitives.
+We use two special Crossplane resources to define and configure new XRs and
+XRCs:
+
+- A `CompositeResourceDefinition` (XRD) _defines_ a new kind of composite
+  resource, including its schema. An XRD may optionally _offer_ a claim.
+- A `Composition` specifies which resources a composite resource will be
+  composed of, and how they should be configured. You can create multiple
+  `Composition` options for each composite resource.
+
+XRDs and Compositions may be packaged as a _configuration_, that may be
+installed to Crossplane. In the examples below we will create a configuration
+that defines a new `CompositePostgreSQLInstance` XR that takes a single
+`storageGB` parameter, and creates a connection `Secret` with keys for
+`username`, `password`, and `endpoint`.
+
+## Create a Configuration Directory
+
+Our configuration will consist of three files:
+
+* `crossplane.yaml` - Metadata about the configuration.
+* `definition.yaml` - The XRD.
+* `composition.yaml` - The Composition.
+
+Crossplane can create a configuration from any directory with a valid
+`crossplane.yaml` metadata file at its root, and one or more XRDs or
+Composition. The directory structure does not matter, as long as the
+`crossplane.yaml` file is at the root. Note that a configuration need not
+contain one XRD and one composition - it could include only an XRD, only a
+composition, several compositions, or any combination thereof.
+
+Before we go any further, we must create a directory in which to build our
+configuration:
+
+```console
+mkdir crossplane-config
+pushd crossplane-config
```

Comment:
```suggestion
cd crossplane-config
```

Is there a specific reason to use `pushd`? It's the first time I saw that command and thought it's something that I should've installed.

Reply:
Mostly because it lets us use `popd` at the end of the guide to return to the directory we started in. I guess `cd ..` would work too. I didn't think `pushd` and `popd` were particularly surprising though.

Reply:
Okay, a brief survey says `cd` is less surprising - I'll update it. :)